src/libcore/cell.rs

```diff
@@ -187,6 +187,29 @@ use ops::{Deref, DerefMut, CoerceUnsized};
 use ptr;
 
 /// A mutable memory location.
+/// # Examples
```

Comment:
yes. you had trailing whitespace; that was the only issue, not the `///`s. In other words

    "///"

not

    "/// "

make sense?